spec-main/api-app-spec.ts

```diff
@@ -1116,7 +1116,8 @@ describe('app module', () => {
     });
   });
 
-  describe('getGPUInfo() API', () => {
+  // FIXME https://github.com/electron/electron/issues/24224
+  ifdescribe(process.platform !== 'linux')('getGPUInfo() API', () => {
```

Comment:
Unable to repro this locally on a vm with `--disable-gpu` as well, should setup our ci container locally and debug further.